extensions/interactions/MusicNotesInput/directives/MusicNotesInput.js

```diff
@@ -371,6 +363,29 @@ oppia.directive('oppiaInteractiveMusicNotesInput', [
                     scope.noteSequence[i].note)),
                 position: 'absolute'
               });
+            if (scope.interactionIsActive) {
+              innerDiv
+                .draggable({
```

Comment:
Move to end of previous line. Deindent following lines by 2.

Reply:
Done